### PR TITLE
BC001 PS01 - first draft

### PR DESCRIPTION
**Pull request by pjanck**

### CODE REVIEW

File: tests/PS01/Dataset/BC001_PS01_reference.ifc [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+#28 = IFCGEOMETRICREPRESENTATIONCONTEXT('3D', 'Model', 3, 1.E-6, #29, #30);
+#29 = IFCAXIS2PLACEMENT3D(#5, #4, #2);
+#30 = IFCDIRECTION((0.,1.));
+
+/* ------- IfcSite is the top most element in the spatial structure ---------------------------- */
+#61 = IFCSITE('27H$neCQf1NwtmczxBInPZ', $, 'Top element in project structure', $, $, #62, $, $, .ELEMENT., $, $, $, $, $);
+/* ------- The IfcSite's placement is the top-most placement ----------------------------------- */
+#62 = IFCLOCALPLACEMENT($, #29);
+/* ------- Project/Spatial structure ----------------------------------------------------------- */
+#63 = IFCRELAGGREGATES('3Hu7f6BmT14B_XS9yS78JZ', $, $, $, #16, (#61));
```
Comment (BenzclyZhang) [inline tests/PS01/Dataset/BC001_PS01_reference.ifc:54]: Do we still set IfcSite as the only top spatial structure element in the file? I guess it is not necessary in this test.

File: tests/PS01/Dataset/README.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+The project defines default units:
+
+- *metre* as `LENGTHUNIT`
+- *square metre* as `AREAUNIT`
+- *cubic metre* as `VOLUMEUNIT`
+- *degree* as `PLANEANGLEUNIT`
+- *kilogram* as `MASSUNIT`
+- *second* as `TIMEUNIT`
+- *EUR* as monetary unit
```
Comment (BenzclyZhang) [inline tests/PS01/Dataset/README.md]: We are in Switzerland :)

File: tests/PS01/Dataset/README.md [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+There is one `IfcBuildingElementProxy`:
+
+- its id is `'3RhbhoXQ53yRzPOejOTUcp'`
+- its name is *Punkte:Referenzpunkt:11547115*
+- its tag is 11547115
+- its geometry is a pyramid turned upside down, with base length 0.5, and height 1.5
+- its position is at (relative to project's origin):
+    - Easting: 700.000970772933
+    - Northing: 1570.0
+    - Elevation: 0.0
```
Comment (BenzclyZhang) [inline tests/PS01/Dataset/README.md:49]: In the original file, there are three IfcBuildingElementProxy as reference point. I think they should all be included in this test in order to validate whether Georeferencing is implemented properly.

File: tests/PS01/Dataset/BC001_PS01_reference.ifc (hunk 1 above)
Comment (pjanck) [reply tests/PS01/Dataset/BC001_PS01_reference.ifc:54]: Well, I believe I need *a* container for the proxy pyramid. I can change this to some other *IfcSpatialElement* - would an *IfcFacility* work?

File: tests/PS01/Dataset/README.md (hunk 2 above)
Comment (pjanck) [reply tests/PS01/Dataset/README.md]: I hoped you'd pick this one up :) Will amend accordingly.

File: tests/PS01/Dataset/README.md (hunk 3 above)
Comment (pjanck) [reply tests/PS01/Dataset/README.md:49]: Ok, I can include these as well. Can you please provide me with a screen shot (similar to the one existing) with them circled?

File: tests/PS01/Dataset/BC001_PS01_reference.ifc (hunk 1 above)
Comment (pjanck) [reply tests/PS01/Dataset/BC001_PS01_reference.ifc:54]: Changed to `IfcFacility` in 797d5d12321585f5311e7ba8e3db63b410a0b474

File: tests/PS01/Dataset/README.md (hunk 2 above)
Comment (pjanck) [reply tests/PS01/Dataset/README.md]: Changed to `CHF` in 797d5d12321585f5311e7ba8e3db63b410a0b474

File: tests/PS01/Dataset/README.md (hunk 3 above)
Comment (pjanck) [reply tests/PS01/Dataset/README.md:49]: All three points included in 797d5d12321585f5311e7ba8e3db63b410a0b474

**Pull request merged**